tools/jenkins/run_full_performance.sh

```diff
@@ -52,4 +52,12 @@ tools/run_tests/run_performance_tests.py \
     --remote_worker_host grpc-performance-server-32core grpc-performance-client-32core grpc-performance-client2-32core \
     || EXIT_CODE=1
 
+# selected scenarios on Windows
```

Comment:
I think it should be safe to run windows benchmarks as part of the main benchmarking job. (it's the list item to run, so if it fails, previous measurement will be uploaded anyway)